src/app/views/authentication/profile/Profile.tsx

```diff
@@ -153,6 +154,9 @@ const SignedInButton = forwardRef<
     <Tooltip content={translateMessage('sign out')} relationship='description'>
       <Button
         aria-label={translateMessage('sign out')}
```

Comment:
this label can be updated to include the signed in username... sth like "abc def sign out"
to fulfill the issue indicated on the video
[Bug 30648 [Graph Explorer] [Application Migration to Fluent UI React v9] [Web] [Profile card] Name of the user who signed in is not announced by screen reader when focus lands on profile card..webm](https://dev.azure.com/microsoftgraph/0985d294-5762-4bc2-a565-161ef349ca3e/_apis/wit/attachments/d8fa845f-64a2-497b-8b98-f78fdff5751e?fileName=Bug%2030648%20%5BGraph%20Explorer%5D%20%5BApplication%20Migration%20to%20Fluent%20UI%20React%20v9%5D%20%5BWeb%5D%20%5BProfile%20card%5D%20Name%20of%20the%20user%20who%20signed%20in%20is%20not%20announced%20by%20screen%20reader%20when%20focus%20lands%20on%20profile%20card..webm&download=true)

Reply:
Oh I got it. Updated.